### ISSUE TITLE
[Feature Request]: Bulk size change

### ISSUE THREAD

**Issue by ogmon1243** [FTBTeam/FTB-Mods-Issues#745]
### Mod

FTB Quests

### Description

It would be nice to be able to choose multiple quests at a time and change all of their sizes in one go. Having to open up the menu and input the new size every time becomes tedious when there are a large number of quests to make smaller or bigger.

Also, I dont know if this is really a problem but maybe add an option to have quests conform to the standard grid placement of quests with size 1. When moving around a quest that is a different size than 1, it can become hard to line the quest up on the same x or y level as a differently sized quest. For instance, when moving around a quest, it could move in increments of 0.375 which cant line up with any other quests that move in increments of 0.5 easily. Alternatively, they could follow divisions of that 0.5 grid movement based on the size of the quest so that they don't overlap. 0.5, 0.25. 0.125 ect.

**Comment by desht** [comment]
Yeah adding more batch settings when multiple quests are selected is something I have been thinking about too.  Size is one, but there are others too (e.g. shape or in fact anything that can be done via the Quick Settings... submenu).  So yes, would be a good QoL addition.

**Comment by desht** [comment]
Bulk quest size changing will be in the next release, but more general Quick Settings... changes will probably wait until MC 1.20 (it needs more fundamental changes to how object configuration is handled in FTB Library).

As for grid size, it's a little tricky.  But you can generally get good results by setting the `Grid Scale` to half the smallest fractional size you use, e.g. if you have quests of size 1.5, set the grid scale to 0.25 (default is 0.5).  Grid Scale can be adjusted via Settings -> Edit File.
